docs/src/manual/constraints.md

```diff
@@ -35,14 +35,15 @@ add!(ch, lc1)
 add!(ch, lc2)
 ```
 
-Linear constraint will affect the sparsity pattern of the stiffness matrix, and as such, it is important to also include 
+Affine constraints will affect the sparsity pattern of the stiffness matrix, and as such, it is important to also include 
 the `ConstraintHandler` as an argument when creating the sparsity pattern:
 
 ```julia
 K = create_sparsity_pattern(dh, ch)
 ```
 
-When solving the system, we account for the affine constraints in the same way as we account for 
+### Solving linear problems
+When solving the system `K⋅a=f`, we account for the affine constraints in the same way as we account for 
```

Comment:
Maybe use math mode here instead?
```suggestion
### Solving linear problems

When solving the system ``\underline{\underline{K}} \underline{a} = \underline{f}`, we account for the affine constraints in the same way as we account for 
```

Reply:
Yes, of course - sorry! 
But shouldn't we use boldface instead of underlines? Seems to be used in most other places.

Reply:
I think we use bold for continuous and underline for discrete variables (see https://ferrite-fem.github.io/Ferrite.jl/stable/examples/hyperelasticity/)

Reply:
👍 , I've homogenized the constraint page to use underlines for discrete variables